.github/workflows/eks-addon-tests.yaml

```diff
@@ -0,0 +1,34 @@
+name: EKS Add-on Tests
+
+on:
+  pull_request:
+  push:
+    branches: [ main ]
+
+jobs:
+  eks-addon-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # TODO: Use ci-matrix.json instead
+        k8s_version: ["1.28", "1.29", "1.30", "1.31", "1.32", "1.33"]
+    
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8
```

Comment:
Might be good to add a comment after the hash specifying which version this is

Reply:
Switched to v5. Not sure why we use the sha in other places